src/components/application_manager/rpc_plugins/rc_rpc_plugin/include/rc_rpc_plugin/rc_consent_manager.h

```diff
@@ -70,7 +70,7 @@ class RCConsentManager {
   virtual rc_rpc_plugin::rc_rpc_types::ModuleConsent GetModuleConsent(
       const std::string& app_id,
       const std::string& mac_address,
-      const rc_rpc_types::ModuleUid& module_id) const = 0;
+      const rc_rpc_types::ModuleUid& module_id) = 0;
```

Comment:
@YarikMamykin is this related to fix?

Reply:
@AKalinich-Luxoft , yes it is related to fix because the way to access LastState dictionary was changed.